src/gam/__init__.py

```diff
@@ -6703,6 +6703,7 @@ def _splitSchemaNameDotFieldName(sn_fn, fnRequired=True):
                 if language.lower() in LANGUAGE_CODES_MAP:
                     lang_item['languageCode'] = LANGUAGE_CODES_MAP[language.lower()]
                 else:
+                    lang_item.pop('preference', None)
```

Comment:
this should not be necessary as preference is not set if customLanguage is set:

https://github.com/jay0lee/google-api-tracker/blob/master/admin-directory_v1.json#L7367